base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -161,69 +169,91 @@ impl ConsensusConstants {
+            max_target_time: 7200,
+            min_difficulty: 1.into(),
+            target_time: 120,
+        });
         ConsensusConstants {
             coinbase_lock_height: 60,
             blockchain_version: 1,
-            future_time_limit: target_block_interval * difficulty_block_window / 20,
-            target_block_interval,
+            future_time_limit: 540,
             difficulty_block_window,
-            difficulty_max_block_interval: target_block_interval * 60,
             max_block_transaction_weight: 19500,
-            pow_algo_count: 1,
             median_timestamp_count: 11,
             emission_initial: 5_538_846_115 * uT,
             emission_decay: 0.999_999_560_409_038_5,
             emission_tail: 1 * T,
-            min_pow_difficulty: (1.into(), 60_000_000.into()),
             max_randomx_seed_height: std::u64::MAX,
             genesis_coinbase_value_offset: 5_539_846_115 * uT - 10_000_100 * uT,
+            proof_of_work: algos,
         }
     }
 
     pub fn localnet() -> Self {
-        let target_block_interval = 120;
         let difficulty_block_window = 90;
+        let mut algos = HashMap::new();
+        algos.insert(PowAlgorithm::Blake, PowAlgorithmConstants {
+            max_target_time: 720,
+            min_difficulty: 1.into(),
+            target_time: 120,
+        });
+        algos.insert(PowAlgorithm::Monero, PowAlgorithmConstants {
+            max_target_time: 720,
+            min_difficulty: 1.into(),
+            target_time: 120,
+        });
         ConsensusConstants {
             coinbase_lock_height: 1,
             blockchain_version: 1,
-            future_time_limit: target_block_interval * difficulty_block_window / 20,
-            target_block_interval,
-            difficulty_max_block_interval: target_block_interval * 6,
+            future_time_limit: 540,
             difficulty_block_window,
             max_block_transaction_weight: 19500,
-            pow_algo_count: 2,
             median_timestamp_count: 11,
             emission_initial: 10_000_000.into(),
             emission_decay: 0.999,
             emission_tail: 100.into(),
-            min_pow_difficulty: (1.into(), 1.into()),
             max_randomx_seed_height: std::u64::MAX,
             genesis_coinbase_value_offset: 0.into(),
+            proof_of_work: algos,
         }
     }
 
     pub fn mainnet() -> Self {
         // Note these values are all placeholders for final values
-        let target_block_interval = 120;
         let difficulty_block_window = 90;
+        let difficulty_block_window = 90;
+        let mut algos = HashMap::new();
+        algos.insert(PowAlgorithm::Blake, PowAlgorithmConstants {
+            max_target_time: 1800,
+            min_difficulty: 60_000_000.into(),
+            target_time: 300,
+        });
+        algos.insert(PowAlgorithm::Monero, PowAlgorithmConstants {
+            max_target_time: 800,
```

Comment:
I believe this value is wrong; should it not be `200 * 6 = 1200`?
(case in point)

Reply:
Hi @mikethetike and @SWvheerden, I see this PR has been merged without fixing the error above or commenting on it?

Reply:
These are place holder mainnet values, and are not used currently. 
If you try to use the mainnet network type, the code gives out a panic

Reply:
The point I am trying to make is that the value is wrong and should be corrected